invenio_search_ui/ext.py

```diff
@@ -31,6 +31,9 @@ def init_app(self, app):
         self.init_config(app)
         app.extensions["invenio-search-ui"] = self
 
+        my_func = app.config.get("SEARCH_UI_SEARCH_FUNCTION", search)
+        blueprint.add_url_rule("/search", "search", view_func=my_func)
```

Comment:
```suggestion
        blueprint.add_url_rule("/search", "search", view_func=search)
```

Reply:
This one looks weird... should it be `search_view` ?

Reply:
```suggestion
        blueprint.add_url_rule("/search", "search", view_func=search_view)
```